build.gradle

```diff
@@ -0,0 +1,77 @@
+buildscript {
+    ext {
+        es_version = System.getProperty("es.version", "6.7.1")
+        kotlin_version = System.getProperty("kotlin.version", "1.3.31")
```

Comment:
Lets add Kotlin Linter as early in the development phase as we can to minimize code style differences.

Reply:
Yep, will do in a follow up PR; there is still quite a bit to add to the gradle configuration which won't be covered by this PR.